src/nvim/ex_docmd.c

```diff
@@ -6788,27 +6801,51 @@ void free_cd_dir(void)
 
 #endif
 
-/*
- * Deal with the side effects of changing the current directory.
- * When "local" is TRUE then this was after an ":lcd" command.
- */
-void post_chdir(int local)
+/// Deal with the side effects of changing the current directory.
+///
+/// @param scope  Scope of the function call (editor, tab or window).
+void post_chdir(int scope)
 {
+  assert(scope >= kCdScopeEditor && scope <= MAX_CD_SCOPE);
+
+  enum cd_scope s = scope; // Treat it as an enum so switch catches unused cases.
```

Comment:
@HiPhish *Push*. You have a bad habit of not providing context, so I cannot say what is the problem.

Most likely you ignored my words about “typedef should be placed before the generated declarations”. At least I have `ex_docmd.h.generated.h` include on the line 23 of `ex_docmd.h` which (given the compiler log) says that you did this.

***KEEP `#ifdef INCLUDE_GENERATED_DECLARATIONS` THE LAST ONE IN THE HEADER FILE.***